targets/TARGET_STM/TARGET_STM32F4/TARGET_STM32F446xE/TARGET_NUCLEO_F446RE/PeripheralPins.c

```diff
@@ -312,14 +312,25 @@ MBED_WEAK const PinMap PinMap_CAN_TD[] = {
 
 //*** QUADSPI ***
 
-MBED_WEAK const PinMap PinMap_QSPI_DATA[] = {
-    {PA_1,      QSPI_1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF9_QSPI)},  // QUADSPI_BK1_IO3
-    {PB_6,      QSPI_1, STM_PIN_DATA(STM_MODE_AF_PP, GPIO_PULLUP, GPIO_AF10_QSPI)},  // QUADSPI_BK1_NCS
```

Comment:
@jeromecoutant This isn't a data pin. Was QSPI always broken on this device?

Reply:
Yes, no QUADSPI_BK1_IO2 pin for STM32F446RE...